vassal-app/src/test/java/VASSAL/build/MockModuleTest.java

```diff
@@ -24,8 +24,8 @@ public void init() throws Exception {
     final DataArchive arch = mock(DataArchive.class);
 
     when(module.getDataArchive()).thenReturn(arch);
-    doNothing().when(module).incorporateSourcesAndListeners();
-    doNothing().when(module).markModuleListeners();
+    //doNothing().when(module).incorporateSourcesAndListeners();
+    //doNothing().when(module).markModuleListeners();
```

Comment:
You can just delete these two lines. (And the import of `doNothing` a bit further up.)

Reply:
done